router/batchrouter/asyncdestinationmanager/snowpipestreaming/channel.go

```diff
@@ -138,6 +138,8 @@ func (m *Manager) createChannel(
 		}
 		m.channelCache.Store(tableName, resp)
 		return resp, nil
+	case internalapi.ErrValidationError:
+		return nil, fmt.Errorf("%w, %w", errAuthz, err)
```

Comment:
Technically speaking, this is an authentication error and not authorization. But I don't think there is a need for that distinction yet.